src/main/scala/cromwell/engine/backend/jes/JesBackendCall.scala

```diff
@@ -72,6 +72,23 @@ class JesBackendCall(val backend: JesBackend,
   def downloadRcFile = authenticateAsUser(workflowDescriptor) { storage => Try(storage.readFile(returnCodeGcsPath)) }
 
   /**
+    * Determines the maximum number of times a call can be started with a Preemptible VM.
+    * Runtime attributes > workflow options > Jes configuration
+    */
+  lazy val maxPreemption = {
```

Comment:
As a heads up, the default runtime attributes in workflow options (coming soon, I promise!) will make the preemption option in the workflow options redundant.